test/end-to-end.js

```diff
@@ -525,6 +530,12 @@ describe('end-to-end', function() {
         .to.be.true;
       done();
     });
+
+    it('comes with legacyExplorer:false flag in config.json', function(done) {
+      var config = fs.readJsonSync(path.resolve(SANDBOX, 'server/config.json'));
+      expect(config.legacyExplorer).to.exist.and.to.be.false;
```

Comment:
Nitpick:

```js
expect(config).to.have.property('legacyExplorer', false)
```

https://github.com/moll/js-must#beware-of-libraries-that-assert-on-property-access